geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -187,27 +194,54 @@ The header can be in a separate file, diff in the `*.html.ejs` file:
 
 The content will be in the `geoportal/{{package}}_geoportal/static/header.html` file.
 
+Print
+-----
 
-Information
+- With the default configuration we can easier switch from local print and mutualised print by setting one
```

Comment:
Suggestion instead of "With the default configuration we can easier switch from local print and mutualised print by setting one..."
-> "You can now switch easily between local print and mutualised print by setting one..."